drivers/gpu/drm/vc4/vc4_hdmi.c

```diff
@@ -3694,6 +3748,8 @@ static int vc4_hdmi_bind(struct device *dev, struct device *master, void *data)
 	if (ret)
 		return ret;
 
+	mutex_init(&vc4_hdmi->update_plugged_status_lock);
```

Comment:
We're missing a matching `mutex_destroy()` in unbind. Admittedly it only does anything if `CONFIG_DEBUG_MUTEXES` is set, but it's nicer to keep it clean.

Reply:
Good point, I'll add that.